spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -113,12 +113,6 @@ def and_I_see_a_success_notice
     expect(page).to have_content("Step by step page was successfully unpublished.")
   end
 
-  def and_I_delete_the_step_by_step_page
```

Comment:
@DilwoarH You can squash this test deletions up into the previous commit.  

Otherwise 👍

Reply:
Done @leenagupte